scheduler/scheduler.go

```diff
@@ -156,7 +156,16 @@ func (s *Scheduler) Schedule(ctx context.Context) error {
 					"expected_duration_span": res.schedulerEvent.ExpectedDuration.String(),
 					"span":     time.Since(distroStartTime).String(),
 					"duration": time.Since(distroStartTime),
-				})
+				}
+				latencies, err := model.AverageTaskLatencyByDistro(d.distroId, time.Minute)
+				if err != nil {
+					grip.Error(errors.Wrap(err, "error getting latencies"))
```

Comment:
can you grip.Error the m object, but add a field to it called "error" where you include the err object instead?